frontend/src/app/components/wp-form-group/wp-attribute-group.component.ts

```diff
@@ -31,22 +31,36 @@ import {I18nService} from 'core-app/modules/common/i18n/i18n.service';
 import {FieldDescriptor, GroupDescriptor} from 'core-components/work-packages/wp-single-view/wp-single-view.component';
 import {WorkPackageResource} from 'core-app/modules/hal/resources/work-package-resource';
 import {EditFormComponent} from "core-app/modules/fields/edit/edit-form/edit-form.component";
+import {UntilDestroyedMixin} from "core-app/helpers/angular/until-destroyed.mixin";
+import {fromEvent} from "rxjs";
+import {debounceTime} from "rxjs/operators";
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/opf/openproject/blob/73d3ddec0f0cb0560924d9a339ae02af8596901c/frontend/src/app/components/wp-form-group/wp-attribute-group.component.ts#L37"></span>Please use <code>ChangeDetectionStrategy.OnPush</code> for this component</td>
    </tr>
  </tbody>
</table>